Author: Michael R. Crusoe <crusoe@debian.org>
Description: Don't build nor test the static or wasm features in Debian
Forwarded: not-needed
--- liblzma-sys.orig/Cargo.toml
+++ liblzma-sys/Cargo.toml
@@ -52,13 +52,8 @@
 default = ["bindgen"]
 fat-lto = []
 parallel = []
-static = []
 thin-lto = []
 uncheck_liblzma_version = []
-wasm = [
-    "static",
-    "bindgen",
-]
 
 [lib]
 name = "liblzma_sys"
